tt-media-server/tests/server_tests/base_test.py

```diff
@@ -0,0 +1,117 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+
+from abc import abstractmethod
+import os
+import asyncio
+import time
+import traceback
+from typing import Any, Dict
+
+from tests.server_tests.test_classes import TestConfig
+
+
+class BaseTest:
```

Comment:
I think you should inherit from ABC (abstract base class):
```suggestion
from abc import ABC, abstractmethod
import os
import asyncio
import time
import traceback
from typing import Any, Dict

from tests.server_tests.test_classes import TestConfig


class BaseTest(ABC):
```